llvm/test/tools/sycl-post-link/spec-constants/multiple-sym-id-usages.ll

```diff
@@ -1,16 +1,14 @@
 ; This test checks that the tool does not crash and removes the unused spec
 ; constant global symbol when it is referenced more than once.
```

Comment:
As I see, you removed FileCheck call here. Is this test now checks only that there is no crash of tool execution?

Reply:
That is correct, since we do not remove any globals anymore, we have nothing to check

Reply:
Then it would be nice to align comment with test purpose.